patches/server/0034-IntelliJ-Remove-Streams.patch

```diff
@@ -0,0 +1,603 @@
+-        private static final Map<String, EnumDirection.EnumAxis> f = (Map) Arrays.stream(EnumDirection.EnumAxis.e).collect(Collectors.toMap(EnumDirection.EnumAxis::b, (enumdirection_enumaxis) -> {
+-            return enumdirection_enumaxis;
+-        }));
++        private static final Map<String, EnumDirection.EnumAxis> f;
++
++        static {
++            Map<String, EnumAxis> map = new HashMap<>();
++            for (EnumAxis enumdirection_enumaxis : EnumAxis.e) {
++                if (map.put(enumdirection_enumaxis.b(), enumdirection_enumaxis) != null) {
++                    throw new IllegalStateException("Duplicate key");
++                }
++            }
++            f = (Map) map;
++        }
++
+         private final String g;
+ 
+         private EnumAxis(String s) {
+diff --git a/src/main/java/net/minecraft/server/EnumProtocol.java b/src/main/java/net/minecraft/server/EnumProtocol.java
+index ec3c8ec6a7c0a835d0bab9872d0ead68b2ff54df..b3cee9bc82aacba99f91ffb4c955ed5321e8250f 100644
+--- a/src/main/java/net/minecraft/server/EnumProtocol.java
++++ b/src/main/java/net/minecraft/server/EnumProtocol.java
+@@ -65,15 +65,17 @@ public enum EnumProtocol {
+             }
+ 
+             EnumProtocol.e[k - -1] = enumprotocol;
+-            enumprotocol.h.forEach((enumprotocoldirection, enumprotocol_a) -> {
+-                enumprotocol_a.a().forEach((oclass) -> {
++            for (Map.Entry<EnumProtocolDirection, a<?>> entry : enumprotocol.h.entrySet()) {
++                EnumProtocolDirection enumprotocoldirection = entry.getKey();
++                a<?> enumprotocol_a = entry.getValue();
++                for (Class<? extends Packet<?>> oclass : enumprotocol_a.a()) {
+                     if (EnumProtocol.f.containsKey(oclass) && EnumProtocol.f.get(oclass) != enumprotocol) {
+                         throw new IllegalStateException("Packet " + oclass + " is already assigned to protocol " + EnumProtocol.f.get(oclass) + " - can't reassign to " + enumprotocol);
+                     } else {
+                         EnumProtocol.f.put(oclass, enumprotocol);
+                     }
+-                });
+-            });
++                }
++            }
+         }
+ 
+     }
+diff --git a/src/main/java/net/minecraft/server/PlayerChunkMap.java b/src/main/java/net/minecraft/server/PlayerChunkMap.java
+index 74998ea20176c2753a0ebb03833f13f3f58a186a..7702169fcf0c2cf72f19ef9472ac435cea05cd21 100644
+--- a/src/main/java/net/minecraft/server/PlayerChunkMap.java
++++ b/src/main/java/net/minecraft/server/PlayerChunkMap.java
+@@ -546,13 +546,13 @@ public class PlayerChunkMap extends IChunkLoader implements PlayerChunk.d {
+         if (viewDistance > 6) {
+             ChunkCoordIntPair front7 = player.getChunkInFront(7);
+             pos.setValues(front7.x << 4, 0, front7.z << 4);
+-            MCUtil.getSpiralOutChunks(pos, 3).forEach(coord -> {
```

Comment:
Not a stream.

https://hg.openjdk.java.net/jdk8/jdk8/jdk/file/tip/src/share/classes/java/util/ArrayList.java#l1226